src/main/java/com/chaos131/poses/MirroredDrivePose.java

```diff
@@ -33,7 +37,7 @@ public DrivePose2024(Pose2d bluePose) {
     }
  </pre>
  */
-public abstract class MirroredDrivePose {
+public class MirroredDrivePose {
```

Comment:
I'm fine with making this non-abstract. 

But just sharing context: we made this as abstract originally to encourage making game specific field pose classes with constructors that set the field width constantly
(though probably should have enforced it through a protected abstract function `getFieldWidthMeters()`)

Reply:
Fair point. I think there was a moment making some unit tests that I just didn't want to be forced to make a new one. But forcing seasonal implementations could be a good idea. We aren't guaranteed every season is mirrored like the last few have been.